src/psyclone/tests/psyir/nodes/structure_member_test.py

```diff
@@ -83,6 +83,28 @@ def test_sm_constructor():
     assert smref.children == []
 
 
+def test_sm_node_equality():
+    ''' Check that two sm nodes are equal appropriately '''
+    kschedule = nodes.KernelSchedule("kname")
+    grid_var = create_structure_symbol(kschedule.symbol_table)
+    assignment = nodes.Assignment(parent=kschedule)
+    assignment2 = nodes.Assignment(parent=kschedule)
+    assignment3 = nodes.Assignment(parent=kschedule)
+    assignment4 = nodes.Assignment(parent=kschedule)
+    grid_ref = nodes.StructureReference.create(grid_var, ['area', 'nx'],
+                                               parent=assignment)
+    grid_ref2 = nodes.StructureReference.create(grid_var, ['area', 'nx'],
+                                               parent=assignment2)
+    grid_ref3 = nodes.StructureReference.create(grid_var, ['area', 'ny'],
+                                               parent=assignment3)
+    grid_ref4 = nodes.StructureReference.create(grid_var, ['aria', 'nx'],
+                                               parent=assignment4)
+
```

Comment:
Pylint say the three parent parameters have "continuation line under-indented for visual indent", they need an extra leading space to match grid_var.

Reply:
Done